WebRoot/public/login.jsp

```diff
@@ -617,9 +617,12 @@ if (application.getInitParameter("offlineMode") != null) {
                                         <td>
                                             <c:import var = "captchaId" url = "${varCaptchaApiUrl}/getCaptchaId"/>
                                             <input id="captchaId" name="captchaId" type="hidden" value="${captchaId}" size="20" maxlength="${domainInfo.webClientMaxInputBufferLength}"/>
-                                            <div width="237" height="50" style="background-color: #F8F8F8" align="center">
+                                            <div style="background-color: #F8F8F8;float: left;width: 70%;height: 50px;" align="left">
                                                <img src="<c:url value='${varCaptchaApiUrl}/captcha/${captchaId}.png'/>" width="150" height="50" name="imageName" alt="image" />
```

Comment:
Related to line 791:
The name 

    name="imageName" 

should be changed so that it identifies the img of CAPTCHA clearly.